### ISSUE TITLE
about the val accuracy

### ISSUE THREAD

**Issue by ZZZZZEROTSU** [hustvl/HAIS#20]
Hi, thanks for your great works! I trained this model and I got the accuracy of  around 0.431, not as high as 0.444 which is provided by the pre-trained model. Also I noticed that the val accurancy is 43.5 in your paper, I just wondering how to get a better result closer to 0.444. Thanks!!!

**Comment by GooseFather990** [comment]
I am not one of the makers but i have seen simular results on other algoritms and most of the time if u increase the number of epochs slightly, u can get closer to the accuracy of the papers.
